gridt/src/app/login/login.page.html

```diff
@@ -27,7 +27,7 @@
               lines="none"
             >
               <ion-label>
-                Should be a valid email address.
```

Comment:
Because we have decided for an email address only login, you can leave email here.